freqtrade/edge/edge_positioning.py

```diff
@@ -159,7 +159,8 @@ def calculate(self, pairs: List[str]) -> bool:
         logger.info(f'Measuring data from {min_date.strftime(DATETIME_PRINT_FORMAT)} '
                     f'up to {max_date.strftime(DATETIME_PRINT_FORMAT)} '
                     f'({(max_date - min_date).days} days)..')
-        headers = ['date', 'buy', 'open', 'close', 'sell', 'high', 'low']
+        #  TODO-lev: Should edge support shorts? needs to be investigated further...
```

Comment:
probably, i don't see why not, I also don't know how to use edge, it's like a trailing stoploss right?

Reply:
no it's not - it's a whole seperate backtesting engine - but one which is VERY limited, cannot (by design) support any callbacks, nor ROI or trailing stop) - so the only sell possibility is via stoploss (static) or sell-signal (this is true for the edge-simulation that's running, anyway).

I'm surely not looking at that part in this pr - mainly because i personally don't see the value in it.

Reply:
it seems like it would be really useful in determining how much leverage to use

Reply:
except that it's not working for about 70% of the strategies, because they either use `custom_sell` for exiting, or rely heavily on ROI / trailing stop - neither of which can work in edge (because it's designed to be 100% vectorized).

Reply:
All I'm saying is that when you apply leverage to a trade, the ideal amount of leverage to apply is an amount that is as high as possible but set's a liquidation price that your trade will not fall to. So maybe in it's current form edge wouldn't be useful, but sounds pretty close to something that can choose the ideal leverage for you

Reply:
I'm not against that - but i 
* don't intend to implement that in this pr
* don't think it's a necessity for short - at least not to start with - so i'd propose to postpone this until "whenever" ... i'd focus on MVP (minimum viable product) with this - not on having 100% of the things there (which is why i'm not keen on supporting all of the different order-types to start with).